pkg/util/strs_test.go

```diff
@@ -27,35 +27,37 @@ package util
 import (
 	"testing"
 
-	"github.com/kasvith/kache/pkg/testsuite"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestSplitSpacesWithQuotes(t *testing.T) {
 	// test for balanced quotes
+	as := assert.New(t)
```

Comment:
Then we can use `assert := testifyAssert.New(t)` which makes code more readiable